py/torch_tensorrt/fx/converters/acc_ops_converters.py

```diff
@@ -675,159 +678,14 @@ def acc_ops_batch_norm(
 
 
 @tensorrt_converter(acc_ops.layer_norm)
-def acc_ops_layer_norm(network, target, args, kwargs, name):
-    input_val = kwargs["input"]
-
-    if not isinstance(input_val, trt.tensorrt.ITensor):
-        raise RuntimeError(
-            f"LayerNorm received input {input_val} that is not part "
-            "of the TensorRT region!"
-        )
-
-    gamma = kwargs["weight"].detach().cpu().float().numpy()
-    gamma_field = trt.PluginField("gamma", gamma, trt.PluginFieldType.FLOAT32)
-    beta = kwargs["bias"].detach().cpu().float().numpy()
-    beta_field = trt.PluginField("beta", beta, trt.PluginFieldType.FLOAT32)
-    eps_field = trt.PluginField(
-        "eps", np.array([kwargs["eps"]], dtype=np.float32), trt.PluginFieldType.FLOAT32
-    )
-    try:
-        normalized_shape = np.array(kwargs["normalized_shape"], dtype=np.int32)
-    except TypeError:
-        _LOGGER.error("Unable to convert normalized_shape to a field, fall back to []")
-        normalized_shape = np.array([], dtype=np.int32)
-
-    normalized_shape_filed = trt.PluginField(
-        "normalized_shape", normalized_shape, trt.PluginFieldType.INT32
-    )
-    field_collection = trt.PluginFieldCollection(
-        [gamma_field, beta_field, eps_field, normalized_shape_filed]
-    )
-
-    try:
-        if network.has_implicit_batch_dimension:
-            plugin = get_trt_plugin("layer_norm", field_collection, "1", "fx2trt")
-        else:
-            plugin = get_trt_plugin("LayerNormDynamic", field_collection, "1", "fx2trt")
-    except AssertionError:
-        _LOGGER.error(
-            "Unable to find layer norm plugin, fall back to TensorRT implementation."
-        )
-        return layer_norm(network, target, args, kwargs, name)
-    layer = network.add_plugin_v2([input_val], plugin)
-    layer.name = name
-    return layer.get_output(0)
-
-
-def layer_norm(
+def acc_ops_layer_norm(
```

Comment:
I think we need a discussion on the development plan for this aten2ait. We hope that acc_ops_converters.py are not changed fundamentally unless it is a feature support.

I suggest a good way as Apurba can work on the `aten_ops_converters.py` and call the converters defined in operators.py. If you can reuse any existing implementations in `acc_ops_converters.py`, that is good and saves your efforts. If you have a better idea to reuse or need to restructure the code, you can create the utility functions in operator.py
So in way, Apurba's development will lay in `aten_ops_converters.py` and 'operator.py'
cc @narendasan @wushirong @yinghai

Reply:
Sure, we can sync up and discuss, that would be good!